docs/en_US/NAS/HardwareAwareNAS.rst

```diff
@@ -0,0 +1,41 @@
+Hardware-aware NAS
+==================
+
+.. contents::
+
+EndToEnd Multi-trial SPOS Demo
+------------------------------
+
+Basically, this demo will select the model whose latency satisfy constraints to train.
+
+To run this demo, first install nn-Meter from source code (currently we haven't released this package, so development installation is required).
```

Comment:
Shall we give a link to nn-Meter here?

Reply:
I'm not sure. My suggestion is that you can do that later when you actually have nn-Meter open sourced.